docs/conf.py

```diff
@@ -48,18 +49,19 @@
 master_doc = 'index'
 
 # General information about the project.
-project = u'ViUR Core'
-copyright = u'2021, Mausbrand Informationssysteme GmbH'
-author = u'Mausbrand Informationssysteme GmbH'
+project = 'ViUR Core'
+copyright = f'{datetime.datetime.now().year}, Mausbrand Informationssysteme GmbH'
+author = 'Mausbrand Informationssysteme GmbH'
 
 # The version info for the project you're documenting, acts as replacement for
 # |version| and |release|, also used in various other places throughout the
 # built documents.
 #
 # The short X.Y version.
-version = u'3.0.0'
+# TODO: Can we import this from core.version?
+version = '3.1.2'
```

Comment:
Hi, since 3.1.2 the version number is hold in viur.core.version as a single string, might this be helpful here?